### PR TITLE
Playlistmanagement

### PR DESCRIPTION
**Pull request by Richa-rani02**
Features Added: 
1.create new playlist
2. two playlist with same name cannot be created
3. delete playlist
4. add videos to playlist
5. delete video from playlist

### CODE REVIEW

File: src/Pages/Playlist/components/Playlistvideo.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import "../Playlist.css";
+import { useParams } from "react-router-dom";
+import { HorizontalCard, Sidebar } from "../../../components/index";
+import { useStateContext } from "../../../context/state-context";
+export const Playlistvideo = () => {
+    const { playlistId } = useParams();
+    const { state: { playlists } } = useStateContext();
+    const playlistList = playlists?.find((play) => play._id === playlistId);
+    console.log(playlistList);
```
Comment (prathmeshjagtap) [inline src/Pages/Playlist/components/Playlistvideo.jsx]: remove console.log

File: src/components/modal/Modal.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
+.popup{
+    /* padding: var(--spacing-1); */
```
Comment (prathmeshjagtap) [inline src/components/modal/Modal.css]: remove commented code if not needed

**Review by prathmeshjagtap — APPROVED** [review]
code is Clean and easily understandable ,

**Review by devananth — APPROVED** [review]
- Code looks Clean.
- Well Stuctured like the way you having separate files for **actions , api's**
- Good Work

**Pull request merged**